cmake/GoogleCloudCpp.cmake

```diff
@@ -40,3 +40,9 @@ FetchContent_MakeAvailable(google-cloud-cpp)
 if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL "14.0.0" )
     target_compile_options(google_cloud_cpp_common PUBLIC -Wno-error=deprecated-pragma)
 endif()
+
+if (WIN32)
+    target_compile_options(google_cloud_cpp_storage PUBLIC /wd4996)
+else()
+    target_compile_options(google_cloud_cpp_storage PUBLIC -Wno-error=deprecated-declarations)
```

Comment:
Would you mind adding a note with an explanation or related error messages?

Reply:
Added in 1a573202fd6a65332b4f7d8e8f3b6b3ecac2673a